client/assets/pages/navbar.scss

```diff
@@ -0,0 +1,25 @@
+@import url('http://fonts.googleapis.com/css?family=Abel');
+@import '../colors.scss';
+
+.navbar {
+  overflow: hidden;
+  box-shadow: 0px 8px 16px 0px rgba(0, 0, 0, 0.2);
+}
+
+.navbar a {
+  font-family: Abel;
+  font-size: 21px;
+  text-align: center;
+  color: map-get($colors, silver);
+  padding: 14px 16px;
+  text-decoration: none;
+}
+
+.navbar__section_bottom {
```

Comment:
This does not follow BEM. Take a refresher on BEM convention: https://github.com/AndelaOSP/Andela-Socials/wiki/BEM-Styleguide